packages/destination-actions/src/destinations/facebook-conversions-api/custom/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ActionDefinition, IntegrationError } from '@segment/actions-core'
+import { action_source, event_id, event_source_url, event_time } from '../fb-capi-properties'
+import { hash_user_data, user_data_field } from '../fb-capi-user-data'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { API_VERSION } from '../constants'
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Custom Event',
+  description: 'Track your own custom events.',
+  defaultSubscription: '',
+  fields: {
+    action_source: { ...action_source, required: true},
+    event_name: {
+      label: 'Event Name',
+      description: 'Send any custom event',
+      type: 'string',
+      required: true,
+      default: {
+        '@path': '$.event'
+      }
+    },
+    event_time: { ...event_time, required: true },
+    user_data: user_data_field,
+    event_id: event_id,
+    event_source_url: event_source_url,
+    custom_data: {
+      label: 'Custom Data',
+      description: 'The custom data object which can be used to pass the custom properties described at: https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data',
```

Comment:
Thanks for doing this @nick-Ag! Small thing, but is it possible to not include the entire URL and instead hyperlink it.

```suggestion
      description: 'The custom data object which can be used to pass the custom properties. See [here](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data) for more information.',
```